scripts/bump-release

```diff
@@ -8,12 +8,18 @@
 #
 # YMMV, no disclaimer or warranty, etc.
 
-if ! [[ "$1" =~ ^v[[:digit:]]+.[[:digit:]]+.[[:digit:]]$ ]]; then 
+if ! [[ "$1" =~ ^v[[:digit:]]+.[[:digit:]]+.[[:digit:]]$ ]]; then
 	echo "Usage: scripts/bump-release <VERSION>"
 	echo "   where VERSION must be vX.Y.Z"
 	exit 255
 fi
 
+# make sure we are running in a toplevel directory
+if ! [[ -d '.git' ]]; then
```

Comment:
How about `[[ $PWD = $(git rev-parse --show-toplevel) ]]`?

Alright, alright, will copy the check from the bad example.